src/Runner/Runner.php

```diff
@@ -169,9 +169,9 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                 $fixer->fix($file, $tokens);
 
                 if ($tokens->isChanged()) {
+                    $appliedFixers[$fixer->getName()] = $tokens->findChangedLines();
```

Comment:
One possible optimization here: we could use `$tokens->findChangedLines()` in lieu of the guarding `$tokens->isChanged()` check, since there will always be changed lines if the tokens collection has changed. Essentially, the `isChanged` guard is obsolete and also inefficient because both methods might loop over the entire token collection.

However, if we want to have an option to disable line numbers the guard would still be needed. Therefore this optimization is a question of whether we should always produce line numbers or not.